app/src/main/res/layout-land/promoted_story_list.xml

```diff
@@ -49,13 +50,21 @@
         android:textSize="14sp" />
     </LinearLayout>
 
-    <androidx.recyclerview.widget.RecyclerView
+    <org.oppia.android.app.home.topiclist.PromotedStoryListView
       android:id="@+id/promoted_story_list_recycler_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:clipToPadding="false"
       android:layout_marginTop="4dp"
       android:overScrollMode="never"
-      android:scrollbars="none" />
+      android:scrollbars="none"
+      android:orientation="horizontal"
+      android:paddingStart="@{viewModel.paddingStart}"
+      android:paddingEnd="@{viewModel.promotedStoryList.size() > 1 ? viewModel.paddingEnd : viewModel.paddingStart}"
+      android:paddingTop="0dp"
+      android:paddingBottom="0dp"
+      app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+      app:reverseLayout="false"
```

Comment:
I think this is also default--do we need to specify this?

Reply:
Same reason, the original implementation had explicitly specified this but it looks like it's not changing the layout at all. Will remove